guide/admin/homeview.md

```diff
@@ -0,0 +1,33 @@
+## User Handbook
+### Changing the Home Page View
```

Comment:
If you consider this from a search perspective, adding "view" here has no value. The bottom line is the page allows for configuring the home page, no matter the technical implementation.

tldr drop "View" 😁